test/groups/group_utils.jl

```diff
@@ -1,31 +1,35 @@
-struct NotImplementedOperation <: AbstractGroupOperation end
+s = @isdefined _group_utils_included
+if !s
+    _group_utils_included = true
```

Comment:
This reminds me of the `#ifndef / #define / #endif` idiom in C and C++ :smile: . We need `#pragma once` in Julia.

Reply:
this is for now also a bit of a quick hack. I learned that Julia 1.9 also did not like multiple includes, but was silent about it, better would be a small test package or such, but for now this reduces the output during testing _tremendously_